cunumeric/module.py

```diff
@@ -2382,7 +2382,7 @@ def compress(c, a, axis=None, out=None):
 
     Parameters
     ----------
-    c : condition, 1-D array of bools
+    c : ndarray[bool]
```

Comment:
FYI note that this is technically incompatible with `numpy` the param name is `condition`

https://github.com/nv-legate/cunumeric/pull/356/files/85b81fe36b50c963d489bf15fca3ab03eca66e76#r872898745